src/renderStatsCard.js

```diff
@@ -1,22 +1,11 @@
-const { kFormatter, fallbackColor } = require("../src/utils");
+const { kFormatter, fallbackColor, FlexLayout } = require("../src/utils");
 const getStyles = require("./getStyles");
 const icons = require("./icons");
 
-const createTextNode = ({
-  icon,
-  label,
-  value,
-  id,
-  index,
-  lineHeight,
-  showIcons,
-}) => {
+const createTextNode = ({ icon, label, value, id, index, showIcons }) => {
   const kValue = kFormatter(value);
   const staggerDelay = (index + 3) * 150;
-  // manually calculating lineHeight based on index instead of using <tspan dy="" />
-  // to fix firefox layout bug
-  const lheight = lineHeight * (index + 1);
-  const translateY = lheight - lineHeight / 2;
```

Comment:
Don't need to calculate these anymore because we can just use FlexLayout function to handle this